src/executables/util_sa.c

```diff
@@ -554,7 +555,7 @@ createdb (UTIL_FUNCTION_ARG * arg)
 	}
       fprintf (stderr, msgcat_message (MSGCAT_CATALOG_UTILS, MSGCAT_UTIL_SET_CREATEDB, CREATEDB_MSG_FAILURE));
       PRINT_AND_LOG_ERR_MSG (msgcat_message (MSGCAT_CATALOG_UTILS, MSGCAT_UTIL_SET_CREATEDB, CREATEDB_MSG_BAD_RANGE),
-			     prm_get_name (PRM_ID_LOG_VOLUME_SIZE), vol_buf, min_buf, max_buf);
+			     prm_get_name (PRM_ID_LOG_VOLUME_SIZE), vol_buf, max_buf);
```

Comment:
Are you suggesting to keep the old message only for log size or for both, log and db volume sizes?